src/language-js/printer-estree.js

```diff
@@ -1159,9 +1159,12 @@ function printPathNoParens(path, options, print, args) {
         printCallArguments(path, options, print),
       ]);
 
-      // We group here when the callee is itself a call expression.
-      // See `isLongCurriedCallExpression` for more info.
-      if (isCallOrOptionalCallExpression(n.callee)) {
+      if (
+        // We group here when the callee is itself a call expression.
+        // See `isLongCurriedCallExpression` for more info.
+        isCallOrOptionalCallExpression(n.callee) ||
+        isBinaryish(path.getParentNode())
```

Comment:
No, we can't tackle it this way:

**Prettier pr-8476**
[Playground link](https://deploy-preview-8476--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAjAlgMwgJygYR3SgHM4cBZCAZwAs4ATABXIAcAbOADwGUBDAO4NyAbQA6UAASS+1ANYAVHBABuCPgCE46gCJxWCBtQBKEAXyjVJAagnSS7dAFtqcAOLsAnq1rUAEnwAXnw4DAByEBCs1ArocLFwEgC6dpIAPgB8ktgQ0gD0eZKQTk4IMKmZknIQYLR8qPiRcW58pQDyUG2oEJwAqlACcHxqFVkOzq4e3r4BwaERUTFxCQDcEiAANCBRMOjQ1MigIcoCTCEIByh87OaeB1uoOHxgcnAwPKzPxCTIMDgArnAtnAnKhGMIGAAZCwkf58MgAMVwTj4MF2pGQID4-xgEE2IFoMCc7AA6rR0PBqJ8wHAeBcKegVBTPJiwNR7iBiK4cDAmE8SCjkJhrq4tgArahcDRPF5vfilSHEOBCkVAkAS3jfTgARX+EHgKvYopAnxw3MxqHqcHY+NYRFgJPQDBgtGQAA4AAxbO00OAkp6sTF2uDctT4gCOevgfMWmNkAFooNoIficHBI+g03z4YKkMKjWrXE50L8AYWtXBdfrlXnVVsYPVHc7XUgAEz1p7oRykRolPiYkMAVnx-1cCnql3zxpUgIAklBhLAeGAiKwYABBBc8GCeTiG1wAXwPQA)
```sh
--parser babel
```

**Input:**
```jsx
bifornCringerMoshedPerplexSawder[
  askTrovenaBeenaDependsRowans +
  glimseGlyphsHazardNoopsTieTie
]
  |> foo  // comment
  |> kochabCooieGameOnOboleUnweave
  |> glimseGlyphsHazardNoopsTieTie;

```

**Output:**
```jsx
bifornCringerMoshedPerplexSawder[
    askTrovenaBeenaDependsRowans + glimseGlyphsHazardNoopsTieTie
  ]
  |> foo  // comment
  |> kochabCooieGameOnOboleUnweave
  |> glimseGlyphsHazardNoopsTieTie;

```

Reply:
I know, I'm fixing it.:+1:

Reply:
done https://github.com/prettier/prettier/pull/8476/commits/d4e5e6cc79be51ed417d603c48b25a8a9c3a67a5